src/modules/user/user.entity.ts

```diff
@@ -21,6 +21,8 @@ export class User extends BaseEntity {
   password: string;
   @Column({ name: 'is_active', default: false })
   isActive: boolean;
+  @Column({ type: 'varchar', name: 'avatar', length: 255 })
+  avatar: string;
```

Comment:
😎 good, can you add this field to data model please

Reply:
Which data model?
It is already in the entity and dto's